oneandone/src/test/java/org/apache/jclouds/oneandone/rest/features/BlockStorageApiLiveTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jclouds.oneandone.rest.features;
+
+import org.apache.jclouds.oneandone.rest.domain.Server;
+import org.apache.jclouds.oneandone.rest.domain.BlockStorage;
+import org.apache.jclouds.oneandone.rest.domain.options.GenericQueryOptions;
+import org.apache.jclouds.oneandone.rest.internal.BaseOneAndOneLiveTest;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+
+
+@Test(groups = "live", testName = "BlockStorageApiLiveTest")
+public class BlockStorageApiLiveTest extends BaseOneAndOneLiveTest {
```

Comment:
as we don't have access to 1&1 endpoint, can you attach the relevant logs of this LiveTest?

Reply:
```
 T E S T S
-------------------------------------------------------
Running org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest
Configuring TestNG with: TestNG652Configurator
Starting test testGet(org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest)
Starting test testListWithOption(org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest)
Starting test testList(org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest)
Starting test testUpdate(org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest)
[pool-1-thread-3] Test testListWithOption(org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest) succeeded: 618ms
Test suite progress: tests succeeded: 1, failed: 0, skipped: 0.
[pool-1-thread-2] Test testList(org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest) succeeded: 841ms
Test suite progress: tests succeeded: 2, failed: 0, skipped: 0.
[pool-1-thread-1] Test testGet(org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest) succeeded: 1732ms
Test suite progress: tests succeeded: 3, failed: 0, skipped: 0.
[pool-1-thread-4] Test testUpdate(org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest) succeeded: 2065ms
Test suite progress: tests succeeded: 4, failed: 0, skipped: 0.
Starting test testAttachBlockStorage(org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest)
[pool-1-thread-5] Test testAttachBlockStorage(org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest) succeeded: 1400ms
Test suite progress: tests succeeded: 5, failed: 0, skipped: 0.
Starting test testDetachServer(org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest)
[pool-1-thread-5] Test testDetachServer(org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest) succeeded: 56239ms
Test suite progress: tests succeeded: 6, failed: 0, skipped: 0.
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 245.489 sec - in org.apache.jclouds.oneandone.rest.features.BlockStorageApiLiveTest

Results :

Tests run: 6, Failures: 0, Errors: 0, Skipped: 0

------------------------------------------------------------------------
BUILD SUCCESS
------------------------------------------------------------------------
Total time: 4:06.543s
Finished at: Mon Jul 09 17:31:48 CEST 2018
Final Memory: 11M/309M
```